### ISSUE TITLE
[QUERY] How to use it?

### ISSUE THREAD

**Issue by M9Apar** [ChargeTimeEU/Java-OCA-OCPP#98]
Hello, i am beginner for this domain, before my query i must say one of best implementation i came across for OCPP.
i successfully compile library into my environment, i am struck at how to use it? i took a reference from example and test module but i can't figure out way to pass IP and PORT for server, so can you please enlighten me with details or it would be great if there is any demo application?

**Comment by TVolden** [comment]
Hi Apar,

Thank you for your kind words.

If your interrested in implementing 1.6, then I can offer this little guide: https://github.com/ChargeTimeEU/Java-OCA-OCPP/wiki/Getting-started

I have some demo apps (they're not much):
Server: https://github.com/ChargeTimeEU/ChargeTime-HQ
Client: https://github.com/ChargeTimeEU/ChargePointSimulator

**Comment by TVolden** [comment]
Did you edit your question after I replied? I didn't notice.

IP and Port are passed as input parameters for both server and client. How to pass IP and port between server and client is up to you. I would recommend storing this information in a configuration file.

In my example I just hardcoded IP to 127.0.0.1 and a port number I picked randomly since I haven't find a standard port for OCPP (help me out here community).

I have some old experimental implementations for the library:
Client simulator: https://github.com/ChargeTimeEU/ChargePointSimulator
Server: https://github.com/ChargeTimeEU/ChargeTime-HQ

Other than that, I would recommend the getting started page.
